2021/scipy/scripts/preprocess.py

```diff
@@ -0,0 +1,71 @@
+import pandas as pd
```

Comment:
I think that having subdirectories is an overkill for such a small tutorial, I'd be in favor of keeping everything in one directory. Just like:
https://github.com/propublica/compas-analysis

One possible exception would be `data` folder (with two files in it). For filenames, I'd loosely follow the conventions here:
https://drivendata.github.io/cookiecutter-data-science/#directory-structure

@fairlearn/fairlearn-maintainers : thoughts?